app/views/setup/_cluster_size_panel.html.slim

```diff
@@ -0,0 +1,19 @@
+.panel.panel-default
+  .panel-heading
+    h3.panel-title Cluster size
+  .panel-body.cluster-size-selector
+    .col-md-8
+      .form-group
+        label for="cloud_cluster_instance_count" Number of instances
+        = form.text_field :instance_count, data: {slider_min: 3, slider_value: 5, slider_max: 50, slider_scale: "logarithmic", slider_tooltip: "always", slider_tooltip_position: "bottom", slider_ticks: [3, 5, 10, 20, 50], slider_ticks_snap_bounds: 0.5}
+    .col-md-2.text-right
+      .form-group.cluster-cpu-count
+        label Total vCPUs
+        h3#cluster-cpu-count = 0
```

Comment:
I wouldn't use a `h3` here. A single `span` with `display: block|inline-block` would do the trick.

Reply:
You can use `.h3` class to reproduce the same font-size and color.

Reply:
What benefit do you see from using
```
span.#cluster-cpu-count.h3 style="display: inline-block" = 0
```
over
```
h3#cluster-cpu-count = 0
```
?

Reply:
Visually speaking nothing changes but semantically speaking it's wrong to use header tags around just because it's a block element and has some style associated with it. Same applies for the `label` element above.

Ideally I would prefer a dedicated class with its own style. Maybe something like `cluster-cpu-count-value` since you already used `cluster-cpu-count` for the div that encapsulates it.

Reply:
Changed in d15b5f90.

Reply:
👍